doc/build-windows.md

```diff
@@ -6,8 +6,11 @@ Below are some notes on how to build Bitcoin Core for Windows.
 Most developers use cross-compilation from Ubuntu to build executables for
 Windows. This is also used to build the release binaries.
 
-Currently only building on Ubuntu Trusty 14.04 is supported.
-Other versions are unsupported or known to be broken (e.g. Ubuntu Xenial 16.04).
+Building on Ubuntu Trusty 14.04 is recommended.
+It is possible to build on Ubuntu Xenial 16.04. Follow the Windows Subsystem for Linux
```

Comment:
Should mention early that 16.04 produces incorrect code, so that people don't get started with it and come to the conclusion later that it's broken.

Reply:
There is no bad code providing the mingw32 x-compiler on wsl/xenial is updated as per the instructions in the doc.